src/results.jl

```diff
@@ -61,6 +75,16 @@ end
 show_result(io::IO, rsp::JSON3.Object) = show(io, TransactionResult(rsp))
 show_result(rsp::JSON3.Object) = show(stdout, TransactionResult(rsp))
 
+function show_result(rsp::TransactionResponse)
+    rsp.metadata === nothing && return
+    rsp.results === nothing && return
+
+    for index in 1:length(rsp.metadata)
+        println(rsp.metadata[index]["relationId"])
+        println(collect(zip(rsp.results[index][2]))[1])
```

Comment:
```suggestion
        println(collect(zip(rsp.results[index]...)))
```
I think the bug here is that you want to print _all_ the columns from the arrow table.
I think this should do it?